src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1758,7 +1758,6 @@ because xbuild doesn't support framework reference assemblies.
 
 <Target Name="_GenerateEnvironmentFiles">
   <ItemGroup>
-    <_GeneratedAndroidEnvironment Include="__XA_PACKAGE_NAMING_POLICY__=$(AndroidPackageNamingPolicy)" />
```

Comment:
I'm not sure how this is relevant to the removal of desktop support

Reply:
This spot seemed to be the only place that actually reads the env var:
https://github.com/dotnet/android/blob/5c5854f4758b7348aee7d9b5926d893e9f6dd73c/src/Mono.Android/Android.Runtime/JNIEnvInit.cs#L142-L147

`$(AndroidPackageNamingPolicy)` is still left in, which embeds a value in `libxamarin-app.so`.